pkgs/applications/audio/aacgain/default.nix

```diff
@@ -1,68 +1,36 @@
-{ lib, stdenv, fetchFromGitLab, fetchpatch }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, autoconf
+, automake
+, libtool
+}:
 
 stdenv.mkDerivation rec {
   pname = "aacgain";
-  version = "1.9.0";
-
-  src = fetchFromGitLab {
-    owner = "mulx";
-    repo = "aacgain";
-    rev = "7c29dccd878ade1301710959aeebe87a8f0828f5";
-    sha256 = "07hl432vsscqg01b6wr99qmsj4gbx0i02x4k565432y6zpfmaxm0";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "dgilman";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
```

Comment:
We should have those libraries in nixpkgs https://github.com/dgilman/aacgain/blob/master/.gitmodules

Reply:
I'm afraid using shared libraries in nixpkgs isn't trivial, as we have something like

```
#include "src/impl.h" //in mp4v2
```

everywhere in the source code. This also prevents dropping autotools.

Reply:
Than I would suggest to at least symlink our versions to receive patches and updates.

Reply:
Done.